### PR TITLE
ECPINT-2783-BM-partial-refunds

### PR DESCRIPTION
**Pull request by AlinAnghelusEndava**
- changed logic to support multiple refunds

### CODE REVIEW

File: Cartridges/bm_checkoutcom/cartridge/static/default/js/core/actions.js [hunk 1]
```diff
@@ -105,20 +105,25 @@ function getTransactionData(members) {
...
+            });
 
             // Add the remaining values
+            jQuery(field1Id).val(transaction.refundable_amount);
             jQuery(field2Id).append(transaction.currency);
             jQuery(field3Id).append(transaction.transaction_id);
             jQuery(field4Id).append(transaction.payment_id);
             jQuery(field5Id).append(transaction.amount + ' ' + transaction.currency);
             jQuery(field6Id).append(transaction.order_no);
+            jQuery(field7Id).append(transaction.refundable_amount + ' ' + transaction.currency);
```
Comment (cko-moho-khaleqi) [inline Cartridges/bm_checkoutcom/cartridge/static/default/js/core/actions.js:125]: Is this the right order for text ? e.g. `10 £` ?

Comment (cko-moho-khaleqi) [reply Cartridges/bm_checkoutcom/cartridge/static/default/js/core/actions.js:125]: I think this should be other way around

**Review by cko-moho-khaleqi — CHANGES_REQUESTED** [review]
please have a look at the comment

File: Cartridges/bm_checkoutcom/cartridge/static/default/js/core/actions.js (hunk 1 above)
Comment (AlinAnghelusEndava) [reply Cartridges/bm_checkoutcom/cartridge/static/default/js/core/actions.js:125]: Left unchanged as we display currency code and not symbol (e.g. 10 USD, not $10)

**Pull request merged**